src/Carbon/Carbon.php

```diff
@@ -1142,6 +1159,10 @@ public function formatLocalized($format)
             $format = preg_replace('#(?<!%)((?:%%)*)%e#', '\1%#d', $format);
         }
 
+        if (static::$utf8) {
+            return utf8_encode(strftime($format, strtotime($this)));
```

Comment:
`strftime($format, strtotime($this))` is repeated twice - maybe extract this to a `$value` variable ?